### PR TITLE
Update librdkafka to v1.6.0-RC2

### LINKED ISSUE #18668: Upgrade librdkafka to 1.5.3 or greater to fix duplicate writes with exactly once semantics and compressed messages
We are experiencing duplicates writes when consuming from Kafka using the Kafka connector while using exactly once semantics with compressed messages. We were able to locate the bug in librdkakfa, the issue https://github.com/edenhill/librdkafka/issues/3020 was resolved in https://github.com/edenhill/librdkafka/releases/tag/v1.5.3 of librdkafka.

Can we change the librdkafka version in Clickhouse?
I couldn't find any way to build Clickhouse with a different librdkafka version and would rather use an official release.

### PR DESCRIPTION
**Pull request by filimonov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Update librdkafka to v1.6.0-RC2. Fixes #18668

Detailed description / Documentation draft:

### CODE REVIEW

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
Test has failed: test_storage_kafka/test.py::test_kafka_rebalance

**Comment by alexey-milovidov** [comment]
Sorry, it will be closed if it does not pass tests.

**Comment by filimonov** [comment]
Have no idea what was the reason of the failure, it looks like some network / CI issue (may be smth happened during iptables manipulations).I've did rebase to fresh master, let's see what will happen this time.

**Comment by alexey-milovidov** [comment]
@filimonov The same test has failed.

**Comment by alexey-milovidov** [comment]
```
WARNING: ThreadSanitizer: data race (pid=7)
  Write of size 4 at 0x7b6000182e10 by thread T65:
    #0 rd_kafka_toppar_op_serve obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_partition.c:2133:39 (clickhouse+0x15b2a54d)
    #1 rd_kafka_op_handle obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_op.c:760:23 (clickhouse+0x15b29507)
    #2 rd_kafka_q_serve obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_queue.c:497:23 (clickhouse+0x15b38fda)
    #3 rd_kafka_thread_main obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka.c:1960:17 (clickhouse+0x15a752b3)
    #4 _thrd_wrapper_function obj-x86_64-linux-gnu/../contrib/librdkafka/src/tinycthread.c:576:9 (clickhouse+0x15b893eb)

  Previous write of size 4 at 0x7b6000182e10 by thread T67:
    #0 rd_kafka_fetch_reply_handle obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_broker.c:4544:47 (clickhouse+0x15aa325c)
    #1 rd_kafka_broker_fetch_reply obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_broker.c:4625:9 (clickhouse+0x15aa2084)
    #2 rd_kafka_buf_callback obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_buf.c:465:17 (clickhouse+0x15aaab34)
    #3 rd_kafka_req_response obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_broker.c:1763:9 (clickhouse+0x15a924d0)
    #4 rd_kafka_recv obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_broker.c:1875:3 (clickhouse+0x15a924d0)
    #5 rd_kafka_transport_io_event obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_transport.c:742:11 (clickhouse+0x15b71bcc)
    #6 rd_kafka_transport_io_serve obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_transport.c:801:17 (clickhouse+0x15b71bcc)
    #7 rd_kafka_broker_ops_io_serve obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_broker.c:3362:21 (clickhouse+0x15aa1078)
    #8 rd_kafka_broker_consumer_serve obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_broker.c:4943:17 (clickhouse+0x15a9cd07)
    #9 rd_kafka_broker_serve obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_broker.c:5048:17 (clickhouse+0x15a9cd07)
    #10 rd_kafka_broker_thread_main obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_broker.c:5205:25 (clickhouse+0x15a96c12)
    #11 _thrd_wrapper_function obj-x86_64-linux-gnu/../contrib/librdkafka/src/tinycthread.c:576:9 (clickhouse+0x15b893eb)

  Location is heap block of size 984 at 0x7b6000182c00 allocated by thread T65:
    #0 calloc <null> (clickhouse+0x89e16b8)
    #1 rd_calloc obj-x86_64-linux-gnu/../contrib/librdkafka/src/rd.h:112:12 (clickhouse+0x15b29966)
    #2 rd_kafka_toppar_new0 obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_partition.c:199:9 (clickhouse+0x15b29966)
    #3 rd_kafka_topic_partition_cnt_update obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_topic.c:757:12 (clickhouse+0x15b6e2df)
    #4 rd_kafka_topic_metadata_update obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_topic.c:1127:10 (clickhouse+0x15b6cb51)
    #5 rd_kafka_topic_new0 obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_topic.c:455:17 (clickhouse+0x15b6c804)
    #6 rd_kafka_toppar_get2 obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_partition.c:425:23 (clickhouse+0x15b2bae6)
    #7 rd_kafka_cgrp_assign obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_cgrp.c:2633:24 (clickhouse+0x15ab944c)
    #8 rd_kafka_cgrp_op_serve obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_cgrp.c:3246:31 (clickhouse+0x15aacd06)
    #9 rd_kafka_op_handle obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_op.c:760:23 (clickhouse+0x15b29507)
    #10 rd_kafka_q_serve obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_queue.c:497:23 (clickhouse+0x15b38fda)
    #11 rd_kafka_thread_main obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka.c:1960:17 (clickhouse+0x15a752b3)
    #12 _thrd_wrapper_function obj-x86_64-linux-gnu/../contrib/librdkafka/src/tinycthread.c:576:9 (clickhouse+0x15b893eb)

  Thread T65 'rdk:m/kafka_con' (tid=74, running) created by thread T3 at:
    #0 pthread_create <null> (clickhouse+0x89e2c8b)
    #1 thrd_create obj-x86_64-linux-gnu/../contrib/librdkafka/src/tinycthread.c:608:21 (clickhouse+0x15b8934e)
    #2 rd_kafka_new obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka.c:2333:7 (clickhouse+0x15a73fd8)
    #3 cppkafka::Consumer::Consumer(cppkafka::Configuration) obj-x86_64-linux-gnu/../contrib/cppkafka/src/consumer.cpp:65:23 (clickhouse+0x15a58ec8)
    #4 void std::__1::allocator<cppkafka::Consumer>::construct<cppkafka::Consumer, cppkafka::Configuration&>(cppkafka::Consumer*, cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:886:28 (clickhouse+0x12e1abd1)
    #5 void std::__1::allocator_traits<std::__1::allocator<cppkafka::Consumer> >::__construct<cppkafka::Consumer, cppkafka::Configuration&>(std::__1::integral_constant<bool, true>, std::__1::allocator<cppkafka::Consumer>&, cppkafka::Consumer*, cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/__memory/allocator_traits.h:519:21 (clickhouse+0x12e1abd1)
    #6 void std::__1::allocator_traits<std::__1::allocator<cppkafka::Consumer> >::construct<cppkafka::Consumer, cppkafka::Configuration&>(std::__1::allocator<cppkafka::Consumer>&, cppkafka::Consumer*, cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/__memory/allocator_traits.h:481:14 (clickhouse+0x12e1abd1)
    #7 std::__1::__shared_ptr_emplace<cppkafka::Consumer, std::__1::allocator<cppkafka::Consumer> >::__shared_ptr_emplace<cppkafka::Configuration&>(std::__1::allocator<cppkafka::Consumer>, cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:2594:9 (clickhouse+0x12e1abd1)
    #8 std::__1::shared_ptr<cppkafka::Consumer> std::__1::allocate_shared<cppkafka::Consumer, std::__1::allocator<cppkafka::Consumer>, cppkafka::Configuration&, void>(std::__1::allocator<cppkafka::Consumer> const&, cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:3360:55 (clickhouse+0x12e1abd1)
    #9 std::__1::shared_ptr<cppkafka::Consumer> std::__1::make_shared<cppkafka::Consumer, cppkafka::Configuration&, void>(cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:3369:12 (clickhouse+0x12e0e472)
    #10 DB::StorageKafka::createReadBuffer(unsigned long) obj-x86_64-linux-gnu/../src/Storages/Kafka/StorageKafka.cpp:425:21 (clickhouse+0x12e0e472)
    #11 DB::StorageKafka::startup() obj-x86_64-linux-gnu/../src/Storages/Kafka/StorageKafka.cpp:304:28 (clickhouse+0x12e0dcca)
    #12 DB::InterpreterCreateQuery::doCreateTable(DB::ASTCreateQuery&, DB::InterpreterCreateQuery::TableProperties const&) obj-x86_64-linux-gnu/../src/Interpreters/InterpreterCreateQuery.cpp:960:10 (clickhouse+0x1252046a)
    #13 DB::InterpreterCreateQuery::createTable(DB::ASTCreateQuery&) obj-x86_64-linux-gnu/../src/Interpreters/InterpreterCreateQuery.cpp:842:20 (clickhouse+0x1251da24)
    #14 DB::InterpreterCreateQuery::execute() obj-x86_64-linux-gnu/../src/Interpreters/InterpreterCreateQuery.cpp:1135:16 (clickhouse+0x1252240f)
    #15 DB::executeQueryImpl(char const*, char const*, DB::Context&, bool, DB::QueryProcessingStage::Enum, bool, DB::ReadBuffer*) obj-x86_64-linux-gnu/../src/Interpreters/executeQuery.cpp:525:32 (clickhouse+0x129c0487)
    #16 DB::executeQuery(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, DB::Context&, bool, DB::QueryProcessingStage::Enum, bool) obj-x86_64-linux-gnu/../src/Interpreters/executeQuery.cpp:868:30 (clickhouse+0x129bede8)
    #17 DB::TCPHandler::runImpl() obj-x86_64-linux-gnu/../src/Server/TCPHandler.cpp:260:24 (clickhouse+0x131f03de)
    #18 DB::TCPHandler::run() obj-x86_64-linux-gnu/../src/Server/TCPHandler.cpp:1417:9 (clickhouse+0x131fd277)
    #19 Poco::Net::TCPServerConnection::start() obj-x86_64-linux-gnu/../contrib/poco/Net/src/TCPServerConnection.cpp:43:3 (clickhouse+0x15c84542)
    #20 Poco::Net::TCPServerDispatcher::run() obj-x86_64-linux-gnu/../contrib/poco/Net/src/TCPServerDispatcher.cpp:112:19 (clickhouse+0x15c84c6e)
    #21 Poco::PooledThread::run() obj-x86_64-linux-gnu/../contrib/poco/Foundation/src/ThreadPool.cpp:199:14 (clickhouse+0x15dedbe1)
    #22 Poco::(anonymous namespace)::RunnableHolder::run() obj-x86_64-linux-gnu/../contrib/poco/Foundation/src/Thread.cpp:55:11 (clickhouse+0x15dec17f)
    #23 Poco::ThreadImpl::runnableEntry(void*) obj-x86_64-linux-gnu/../contrib/poco/Foundation/src/Thread_POSIX.cpp:345:27 (clickhouse+0x15dea987)

  Thread T67 'rdk:broker1' (tid=76, running) created by thread T3 at:
    #0 pthread_create <null> (clickhouse+0x89e2c8b)
    #1 thrd_create obj-x86_64-linux-gnu/../contrib/librdkafka/src/tinycthread.c:608:21 (clickhouse+0x15b8934e)
    #2 rd_kafka_broker_add obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_broker.c:5484:6 (clickhouse+0x15a963fb)
    #3 rd_kafka_brokers_add0 obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_broker.c:5886:14 (clickhouse+0x15a984a5)
    #4 rd_kafka_new obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka.c:2366:7 (clickhouse+0x15a74106)
    #5 cppkafka::Consumer::Consumer(cppkafka::Configuration) obj-x86_64-linux-gnu/../contrib/cppkafka/src/consumer.cpp:65:23 (clickhouse+0x15a58ec8)
    #6 void std::__1::allocator<cppkafka::Consumer>::construct<cppkafka::Consumer, cppkafka::Configuration&>(cppkafka::Consumer*, cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:886:28 (clickhouse+0x12e1abd1)
    #7 void std::__1::allocator_traits<std::__1::allocator<cppkafka::Consumer> >::__construct<cppkafka::Consumer, cppkafka::Configuration&>(std::__1::integral_constant<bool, true>, std::__1::allocator<cppkafka::Consumer>&, cppkafka::Consumer*, cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/__memory/allocator_traits.h:519:21 (clickhouse+0x12e1abd1)
    #8 void std::__1::allocator_traits<std::__1::allocator<cppkafka::Consumer> >::construct<cppkafka::Consumer, cppkafka::Configuration&>(std::__1::allocator<cppkafka::Consumer>&, cppkafka::Consumer*, cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/__memory/allocator_traits.h:481:14 (clickhouse+0x12e1abd1)
    #9 std::__1::__shared_ptr_emplace<cppkafka::Consumer, std::__1::allocator<cppkafka::Consumer> >::__shared_ptr_emplace<cppkafka::Configuration&>(std::__1::allocator<cppkafka::Consumer>, cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:2594:9 (clickhouse+0x12e1abd1)
    #10 std::__1::shared_ptr<cppkafka::Consumer> std::__1::allocate_shared<cppkafka::Consumer, std::__1::allocator<cppkafka::Consumer>, cppkafka::Configuration&, void>(std::__1::allocator<cppkafka::Consumer> const&, cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:3360:55 (clickhouse+0x12e1abd1)
    #11 std::__1::shared_ptr<cppkafka::Consumer> std::__1::make_shared<cppkafka::Consumer, cppkafka::Configuration&, void>(cppkafka::Configuration&) obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:3369:12 (clickhouse+0x12e0e472)
    #12 DB::StorageKafka::createReadBuffer(unsigned long) obj-x86_64-linux-gnu/../src/Storages/Kafka/StorageKafka.cpp:425:21 (clickhouse+0x12e0e472)
    #13 DB::StorageKafka::startup() obj-x86_64-linux-gnu/../src/Storages/Kafka/StorageKafka.cpp:304:28 (clickhouse+0x12e0dcca)
    #14 DB::InterpreterCreateQuery::doCreateTable(DB::ASTCreateQuery&, DB::InterpreterCreateQuery::TableProperties const&) obj-x86_64-linux-gnu/../src/Interpreters/InterpreterCreateQuery.cpp:960:10 (clickhouse+0x1252046a)
    #15 DB::InterpreterCreateQuery::createTable(DB::ASTCreateQuery&) obj-x86_64-linux-gnu/../src/Interpreters/InterpreterCreateQuery.cpp:842:20 (clickhouse+0x1251da24)
    #16 DB::InterpreterCreateQuery::execute() obj-x86_64-linux-gnu/../src/Interpreters/InterpreterCreateQuery.cpp:1135:16 (clickhouse+0x1252240f)
    #17 DB::executeQueryImpl(char const*, char const*, DB::Context&, bool, DB::QueryProcessingStage::Enum, bool, DB::ReadBuffer*) obj-x86_64-linux-gnu/../src/Interpreters/executeQuery.cpp:525:32 (clickhouse+0x129c0487)
    #18 DB::executeQuery(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, DB::Context&, bool, DB::QueryProcessingStage::Enum, bool) obj-x86_64-linux-gnu/../src/Interpreters/executeQuery.cpp:868:30 (clickhouse+0x129bede8)
    #19 DB::TCPHandler::runImpl() obj-x86_64-linux-gnu/../src/Server/TCPHandler.cpp:260:24 (clickhouse+0x131f03de)
    #20 DB::TCPHandler::run() obj-x86_64-linux-gnu/../src/Server/TCPHandler.cpp:1417:9 (clickhouse+0x131fd277)
    #21 Poco::Net::TCPServerConnection::start() obj-x86_64-linux-gnu/../contrib/poco/Net/src/TCPServerConnection.cpp:43:3 (clickhouse+0x15c84542)
    #22 Poco::Net::TCPServerDispatcher::run() obj-x86_64-linux-gnu/../contrib/poco/Net/src/TCPServerDispatcher.cpp:112:19 (clickhouse+0x15c84c6e)
    #23 Poco::PooledThread::run() obj-x86_64-linux-gnu/../contrib/poco/Foundation/src/ThreadPool.cpp:199:14 (clickhouse+0x15dedbe1)
    #24 Poco::(anonymous namespace)::RunnableHolder::run() obj-x86_64-linux-gnu/../contrib/poco/Foundation/src/Thread.cpp:55:11 (clickhouse+0x15dec17f)
    #25 Poco::ThreadImpl::runnableEntry(void*) obj-x86_64-linux-gnu/../contrib/poco/Foundation/src/Thread_POSIX.cpp:345:27 (clickhouse+0x15dea987)

SUMMARY: ThreadSanitizer: data race obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_partition.c:2133:39 in rd_kafka_toppar_op_serve
==================
```

**Comment by alexey-milovidov** [comment]
@filimonov They have 1.6.0 RC1, try to update first.

**Comment by alexey-milovidov** [comment]
https://github.com/edenhill/librdkafka/issues/3215

**Comment by alexey-milovidov** [comment]
@filimonov

How to read the results of integration tests:
- download and unpack the tar;
- read logs and stderr.

**Comment by filimonov** [comment]
Yep, find it already too. Will try 1.6

**Comment by filimonov** [comment]
Seems cmake scripts should be adjusted this time.

File: contrib/librdkafka-cmake/CMakeLists.txt [hunk 1]
```diff
@@ -2,34 +2,35 @@ set(RDKAFKA_SOURCE_DIR ${ClickHouse_SOURCE_DIR}/contrib/librdkafka/src)
...
-#  ${RDKAFKA_SOURCE_DIR}/regexp.c
   ${RDKAFKA_SOURCE_DIR}/rdaddr.c
   ${RDKAFKA_SOURCE_DIR}/rdavl.c
   ${RDKAFKA_SOURCE_DIR}/rdbuf.c
   ${RDKAFKA_SOURCE_DIR}/rdcrc32.c
   ${RDKAFKA_SOURCE_DIR}/rddl.c
   ${RDKAFKA_SOURCE_DIR}/rdfnv1a.c
+  ${RDKAFKA_SOURCE_DIR}/rdgz.c
   ${RDKAFKA_SOURCE_DIR}/rdhdrhistogram.c
-  ${RDKAFKA_SOURCE_DIR}/rdkafka.c
```
Comment (filimonov) [inline contrib/librdkafka-cmake/CMakeLists.txt:18]: ordered them by name (just like `ls` returns) - this way easier to diff.

**Comment by alexey-milovidov** [comment]
@filimonov 

The test `test_storage_kafka/test.py::test_kafka_duplicates_when_commit_failed` is failed.

**Comment by filimonov** [comment]
Checked 
logs from thread, see some assert from librdkafka 
```
*** ../contrib/librdkafka/src/rdkafka_cgrp.c:2824:rd_kafka_cgrp_op_handle_OffsetCommit: assert: rk->rk_consumer.wait_commit_cnt > 0 ***
```


```
2021.01.16 01:40:43.935422 [ 29 ] {} <Error> StorageKafka (kafka): Exception during commit attempt: Local: Waiting for coordinator
2021.01.16 01:40:43.937360 [ 63 ] {} <Fatal> BaseDaemon: ########################################
2021.01.16 01:40:43.937548 [ 63 ] {} <Fatal> BaseDaemon: (version 21.1.1.5689, build id: D4E97207D34D81D9D96D33F98FFEE664F640697D) (from thread 56) (no query) Received signal Aborted (6)
2021.01.16 01:40:43.937648 [ 63 ] {} <Fatal> BaseDaemon: 
2021.01.16 01:40:43.937891 [ 63 ] {} <Fatal> BaseDaemon: Stack trace: 0x7f7ce201818b 0x7f7ce1ff7859 0x89e97ae 0x15a27c24 0x15a50d91 0x15a252f5 0x15a26009 0x15ad92ec 0x15aea75b 0x15a2a424 0x15b4f9fc 0x89e3fad 0x7f7ce21d3609 0x7f7ce20f4293
2021.01.16 01:40:43.938148 [ 63 ] {} <Fatal> BaseDaemon: 5. raise @ 0x4618b in /usr/lib/x86_64-linux-gnu/libc-2.31.so
2021.01.16 01:40:43.938282 [ 63 ] {} <Fatal> BaseDaemon: 6. abort @ 0x25859 in /usr/lib/x86_64-linux-gnu/libc-2.31.so
2021.01.16 01:40:44.004745 [ 63 ] {} <Fatal> BaseDaemon: 7. abort @ 0x89e97ae in /usr/bin/clickhouse
2021.01.16 01:40:44.053597 [ 63 ] {} <Fatal> BaseDaemon: 8. ./obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka.c:4355: ? @ 0x15a27c24 in /usr/bin/clickhouse
2021.01.16 01:40:44.104204 [ 63 ] {} <Fatal> BaseDaemon: 9. ./obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_cgrp.c:2824: ? @ 0x15a50d91 in /usr/bin/clickhouse
2021.01.16 01:40:44.152829 [ 63 ] {} <Fatal> BaseDaemon: 10. ./obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdatomic.h:86: rd_kafka_buf_callback @ 0x15a252f5 in /usr/bin/clickhouse
2021.01.16 01:40:44.201539 [ 63 ] {} <Fatal> BaseDaemon: 11. ./obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_buf.c:0: rd_kafka_buf_handle_op @ 0x15a26009 in /usr/bin/clickhouse
2021.01.16 01:40:44.250367 [ 63 ] {} <Fatal> BaseDaemon: 12. ./obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_op.c:0: rd_kafka_op_handle @ 0x15ad92ec in /usr/bin/clickhouse
2021.01.16 01:40:44.299207 [ 63 ] {} <Fatal> BaseDaemon: 13. ./obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka_queue.c:500: rd_kafka_q_serve @ 0x15aea75b in /usr/bin/clickhouse
2021.01.16 01:40:44.348605 [ 63 ] {} <Fatal> BaseDaemon: 14. ./obj-x86_64-linux-gnu/../contrib/librdkafka/src/rdkafka.c:2005: rd_kafka_thread_main @ 0x15a2a424 in /usr/bin/clickhouse
2021.01.16 01:40:44.397792 [ 63 ] {} <Fatal> BaseDaemon: 15. ./obj-x86_64-linux-gnu/../contrib/librdkafka/src/tinycthread.c:586: _thrd_wrapper_function @ 0x15b4f9fc in /usr/bin/clickhouse
2021.01.16 01:40:44.457518 [ 63 ] {} <Fatal> BaseDaemon: 16. __tsan_thread_start_func @ 0x89e3fad in /usr/bin/clickhouse
2021.01.16 01:40:44.457729 [ 63 ] {} <Fatal> BaseDaemon: 17. start_thread @ 0x9609 in /usr/lib/x86_64-linux-gnu/libpthread-2.31.so
2021.01.16 01:40:44.457867 [ 63 ] {} <Fatal> BaseDaemon: 18. __clone @ 0x122293 in /usr/lib/x86_64-linux-gnu/libc-2.31.so
2021.01.16 01:40:45.566231 [ 63 ] {} <Fatal> BaseDaemon: Calculated checksum of the binary: DE711F4CBCF29E50994968EDB4981AA8. There is no information about the reference checksum.
2021.01.16 01:40:55.488586 [ 1 ] {} <Fatal> Application: Child process was terminated by signal 6.
```

**Comment by alexey-milovidov** [comment]
It's another bug in `rdkafka` library, could you please make a report to them until version 1.6.0 become released?

**Comment by filimonov** [comment]
https://github.com/edenhill/librdkafka/issues/3217

**Comment by filimonov** [comment]
Ok. It seems now Kafka tests are green.

Several other failed:


Test name | Test status | Test time, sec.
-- | -- | --
test_adaptive_granularity/test.py::test_version_single_node_update[n1-tables1] | FAIL | 51.88
test_adaptive_granularity/test.py::test_mixed_granularity_single_node[node0] | FAIL | 40.93
test_adaptive_granularity/test.py::test_mixed_granularity_single_node[node1] | FAIL | 26.99
test_adaptive_granularity/test.py::test_version_single_node_update[n0-tables0] | FAIL | 23.97
test_default_compression_codec/test.py::test_default_codec_version_update | FAIL | 60.81
test_version_update_after_mutation/test.py::test_upgrade_while_mutation | FAIL | 24.64
test_version_update_after_mutation/test.py::test_mutate_and_upgrade | FAIL | 58.59

**Comment by alexey-milovidov** [comment]
> Cannot reserve 1.00 MiB, not enough space.

Not enough disk space in CI.

**Pull request merged**